src/Examples/Flowsheets/HDA_flowsheet_with_costing.ipynb

```diff
@@ -0,0 +1,1598 @@
+    "- Fomulate and solve a process economics optimization problem\n",
+    "    - Defining an objective function\n",
+    "    - Setting variable bounds\n",
+    "    - Adding additional constraints \n",
+    "\n",
+    "\n",
+    "## Problem Statement\n",
+    "\n",
+    "Hydrodealkylation is a chemical reaction that often involves reacting\n",
+    "an aromatic hydrocarbon in the presence of hydrogen gas to form a\n",
+    "simpler aromatic hydrocarbon devoid of functional groups. In this\n",
+    "example, toluene will be reacted with hydrogen gas at high temperatures\n",
+    " to form benzene via the following reaction:\n",
+    "\n",
+    "**C<sub>6</sub>H<sub>5</sub>CH<sub>3</sub> + H<sub>2</sub> → C<sub>6</sub>H<sub>6</sub> + CH<sub>4</sub>**\n",
+    "\n",
+    "\n",
+    "This reaction is often accompanied by an equilibrium side reaction\n",
+    "which forms diphenyl, which we will neglect for this example.\n",
+    "\n",
+    "This example is based on the 1967 AIChE Student Contest problem as\n",
+    "present by Douglas, J.M., Chemical  Design of Chemical Processes, 1988,\n",
+    "McGraw-Hill.\n",
+    "\n",
+    "Users may refer to the prior examples linked at the top of this notebook for detailed process descriptions of the two HDA configurations. As before, the properties required for this module are defined in\n",
+    "\n",
+    "- `hda_ideal_VLE.py`\n",
+    "- `idaes.models.properties.activity_coeff_models.BTX_activity_coeff_VLE`\n",
+    "- `hda_reaction_kinetic.py`\n",
+    "\n",
+    "Additionally, we will be importing externally-defined flowsheets for the two HDA configurations from\n",
+    "\n",
+    "- `hda_flowsheets.py`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Import and run HDA Flowsheets\n",
+    "First, we will generate solved flowsheets for each HDA model. The external scripts build and set inputs for the flowsheets, initialize unit models and streams, and solve the flowsheets before returning the model objects. Note that the HDA flowsheets contain all unit models and stream connections, and no costing equations:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# source file for external flowsheets\n",
+    "from hda_flowsheets import hda_with_flash, hda_with_distillation"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Building flowsheet...\n",
+      "\n",
+      "Setting inputs...\n",
+      "\n",
+      "Initializing flowsheet...\n",
+      "\n",
+      "Limiting Wegstein tear to 5 iterations to obtain initial solution, if not converged IPOPT will pick up and continue.\n",
+      "\n",
+      "WARNING: Wegstein failed to converge in 5 iterations\n",
```

Comment:
You need to have a text box before this warning/reminding the user that they will see this and that it is expected and OK (as long as the final solve works).